templates/shortcodes/references.md

```diff
@@ -20,13 +21,14 @@
 [mythic_beasts]: https://www.mythic-beasts.com/
 [mythic_beasts_donation]: https://www.mythic-beasts.com/blog/2021/10/06/improving-the-world-bit-by-expensive-bit/
 [openstreetmap]: https://www.openstreetmap.org/about "OpenStreetMap"
+[public_transport_validator]: https://cdn.organicmaps.app/subway/
+[reddit]: https://www.reddit.com/r/organicmaps/
 [repology]: https://repology.org/project/organicmaps/versions
 [telegram]: https://t.me/OrganicMapsApp
+[telegram_chat]: https://t.me/OrganicMaps
```

Comment:
Please keep Telegram link in `_index.md` for localization.

Reply:
It will be duplicated almost everywhere. Supporting many channels is time-consuming. Forums can replace them btw. 
What's the point?

Reply:
For localizing `t.me/OrganicMapsRu`.

Reply:
Exceptions like Ru or others can be added locally on the corresponding pages, no?

Reply:
Yes, I agree. Fixed.